db/queries/assessment_records/queries.py

```diff
@@ -273,14 +274,20 @@ def get_application_jsonb_blob(application_id: str) -> dict:
     return application_json
 
 
-def update_status_to_completed(application_id):
-    from flask import current_app
-    current_app.logger.info(
-             f"Updating application status to COMPLETED")
-    db.session.query(AssessmentRecord)\
+def update_status(application_id, status):
+    if status == "COMPLETED":
+        current_app.logger.info("Updating application status to COMPLETED")
+        db.session.query(AssessmentRecord)\
```

Comment:
We are making very poor use of the argument here. Could we have a single query and pass in the status to it?

We can lookup the status from String using array syntax e.g. `Status['COMPLETED']`

If we want to constrain the input we could check at the start of the function e.g.
```
if status not in ['COMPLETED', 'QA_COMPLETE']:
    abort(400, "Incorrect Status given")
```

Reply:
done!